src/SourceBrowser.Site/Scripts/upload.js

```diff
@@ -0,0 +1,29 @@
+﻿$(document).ready(function () {
+    resetUI();
+    registerButtonClickEvent();
+});
+
+function registerButtonClickEvent() {
+    getButtonFromDom().click(prepareForChange);
```

Comment:
Hmm... I'll play around with that.  I thought the page still came up, but I admit I didn't dive too deeply into the controller after I saw the redirect happen.  Thanks!